frontend/src/store/editorSlice.ts

```diff
@@ -1,27 +1,39 @@
-import { createSlice } from "@reduxjs/toolkit";
 import type { PayloadAction } from "@reduxjs/toolkit";
-import { RootState } from "./store";
+import { createSlice } from "@reduxjs/toolkit";
+import { EditorView } from "codemirror";
 import * as yorkie from "yorkie-js-sdk";
-import { YorkieCodeMirrorDocType, YorkieCodeMirrorPresenceType } from "../utils/yorkie/yorkieSync";
 import { ShareRole } from "../utils/share";
-import { EditorView } from "codemirror";
+import { YorkieCodeMirrorDocType, YorkieCodeMirrorPresenceType } from "../utils/yorkie/yorkieSync";
+import { RootState } from "./store";
+
+export enum EditorModeType {
+	edit = "edit",
+	both = "both",
+	read = "read",
+}
+
+export enum CodeKeyType {
+	sublime = "sublime",
+	vim = "vim",
+}
```

Comment:
Could you change the enum members to use capital letters?

For example, refer to the code below:
```typescript
export enum EditorModeType {
	EDIT = "edit",
	BOTH = "both",
	READ = "read",
}
```